### ISSUE TITLE
re-introduce accounts.mnemonic method

### ISSUE THREAD

**Issue by iamdefinitelyahuman** [iamdefinitelyahuman/brownie#59]
In making brownie install via `pip` I came up with dependency conflicts and realized that the two1 library handling mnemonics hasn't been maintained in over 2 years.

With commit 616122f66507319d493d79cac2e904ab832deecb I've temporarily removed the `accounts.mnemonic` method.  I will reintroduce it once ethereum/eth-account#33 is merged, or if another solution presents itself.

**Comment by iamdefinitelyahuman** [comment]
Another possible solution:

https://github.com/vergl4s/ethereum-mnemonic-utils

**Comment by iamdefinitelyahuman** [comment]
requirements If this is to happen via `eth-account` - https://github.com/esPass/pretix-eth-payment-plugin/issues/64#issuecomment-512384664

**Comment by iamdefinitelyahuman** [comment]
New issue: #334
